src/main/java/com/exadel/discountwebapp/user/entity/User.java

```diff
@@ -0,0 +1,68 @@
+package com.exadel.discountwebapp.user.entity;
+
+import com.exadel.discountwebapp.role.entity.Role;
+import com.exadel.discountwebapp.userdiscount.entitiy.UserDiscount;
+import com.exadel.discountwebapp.location.entity.Location;
+
+import lombok.*;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import javax.persistence.*;
+
+@Data
+@Entity
+@Builder
+@Table(name = "user")
+@NoArgsConstructor
+@AllArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
+public class User {
+
+    @Id
+    @EqualsAndHashCode.Exclude
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "usr_id")
+    private Long id;
+
+    @Column(name = "usr_name", length = 50, nullable = false)
+    private String name;
+
+    @Column(name = "usr_surname", length = 50, nullable = false)
+    private String surname;
+
+    @Column(name = "usr_email", length = 255, unique = true, nullable = false)
+    private String email;
+
+    @EqualsAndHashCode.Exclude
```

Comment:
password is a part of user identity, so we should not exclude it from equals/hashcode